### ISSUE TITLE
Push TX??

### ISSUE THREAD

**Issue by RogueAnon** [biteasy/blockchain-api#15]
Now that we have the script_pub_key in a workable format, all we need is PushTX and we can drop the buggy blockchain.info forever. Any plans for this?

**Comment by biteasy** [comment]
Hehe, how did you find script_pub_key, we just got it up and haven't finished few last tests :)

Well, lots of people have requested PushTX so it's in our highest priorities to deliver within the next few days. We will make an announcement soon. 

I wanted to ask you; How is your experience with site/api so far?

**Comment by RogueAnon** [comment]
I found it using your sandbox tool earlier, I haven't tried it with my app yet.
I'm really happy with your API, it's making my life a lot easier,
keep up the good work!

**Comment by biteasy** [comment]
Thanks for your nice words! We have a lot of things cooking and we are trying to deliver as soon as possible :)

Stay tuned for that push tx feature!

**Comment by schildbach** [comment]
What does "push tx" mean? Would it be a transaction submission feature? I'd be interested in that as well as it would make sense to push a paper wallet redeeming transaction immediately to your server, such that the unspent outputs API is updated almost immediatly.

**Comment by biteasy** [comment]
Yeah it's a transaction submission feature which other sites name it "push tx". It's in our high priority plans to add this.

**Comment by dcousens** [comment]
@biteasy is this feature supported yet?
Not being able to push transactions is a limiting factor in using your service.

**Comment by biteasy** [comment]
@dcousens , not yet but it will be scheduled for implementation very soon. We will post here when we have an exact date.

**Comment by Ant1234ac13** [comment]
What are a list of basic commands?
Can you explain this system for me easy to follow?
